pkg/schema/schema.go

```diff
@@ -167,6 +181,30 @@ func newCollectionItemValueType(collectionItemValue interface{}, position *filep
 	return nil, fmt.Errorf("Collection item type did not match any known types")
 }
 
+func processTypeAnnotation(templateAnnotations template.NodeAnnotations) (bool, error) {
+	if ann, anyType := templateAnnotations[AnnotationSchemaType]; anyType {
+		kwargName, err := core.NewStarlarkValue(ann.Kwargs[0][0]).AsString()
```

Comment:
We should do some validation that `ann.Kwargs[0][0]` exists. If someone accidentally types `#@schema/type any` instead of `#@schema/type any=True` this panics.

Reply:
I found that that panic when providing `#@schema/type any` comes from `ann.Kwargs[0]` being out of range, thus I think it is sufficient enough to check the length of `ann.Kwargs`.

However, one thing that I discovered that I could not figure out how to avoid  were these scenerios: 
- `#@schema/type any=something` gives this unhelpful error:
```
- undefined: something
    schema.yml:3 | #@schema/type any=something
````
however, we get an actual schema error message is `something` is in quotes.
- `#@schema/type any True` gives this unhelpful error:
```
- got identifier, want ','
    schema.yml:3 | #@schema/type any True
```
which seem to error during the EvalYaml call.
It doesn't feel like this issue is in scope due to the fact that I have seen this error before when using other annotations.

Reply:
Yeah I agree, that seems out of scope since those are similar errors to what we see in other areas of code now. 👍